ldclient.go

```diff
@@ -327,6 +330,14 @@ func MakeCustomClient(sdkKey string, config Config, waitFor time.Duration) (*LDC
 	if waitFor > 0 && client.dataSource != datasource.NewNullDataSource() {
 		loggers.Infof("Waiting up to %d milliseconds for LaunchDarkly client to start...",
 			waitFor/time.Millisecond)
+
+		// If you use a long duration and wait for the timeout, then any network delays will cause
+		// your application to wait a long time before continuing execution.
+		if waitFor.Seconds() > HighWaitForSeconds {
+			loggers.Warnf("Make Client was called with a timeout greater than %d. "+
+				"We recommend a timeout of less than %d seconds", HighWaitForSeconds)
```

Comment:
I suspect this will only substitute the seconds for %d once, you might have to add the parameter twice

Reply:
You are right, I tested in the Go playground but was only using one %d before. This is a case where writing a simple string is probably better than the format. I added the argument now.